src/frontend/src/components/common/Modal/_modal.scss

```diff
@@ -1,21 +1,46 @@
 .modal {
-  position: fixed;
+  position: flex;
```

Comment:
`position: flex`? 🤔  Guess that's not even a property, or I am not aware of at least

Reply:
yes, you are right